cmd/dp-import-tracker/main.go

```diff
@@ -226,19 +224,28 @@ func manageActiveInstanceEvents(
 	instanceLoopDoneChan chan bool,
 	store store.Storer,
 	dataImportCompleteProducer kafka.Producer,
+	checkCompleteInterval time.Duration,
 ) {
 
 	// inform main() when we have stopped processing events
 	defer close(instanceLoopDoneChan)
 
 	trackedInstances := make(trackedInstanceList)
-	if _, err := trackedInstances.getInstanceList(ctx, datasetAPI); err != nil {
-		logFatal("could not obtain initial instance list", err, nil)
+	for i := 1; i <= 10; i++ {
```

Comment:
We should consider using RCHTTP for doing these retries? If not also consider making the parameters configurable?

Reply:
this was mostly a "be more persistent at startup" to get the instance list than rchttp allows

if connections fail instantly, perhaps because the dataset-api is down, then we need to backoff and retry more persistently (linearly) than rchttp's exponentially-backing off algorithm

I'll add configuration for it, though

Reply:
done